### PR TITLE
[baxtereus] Pass start-time to start-offset-time of :angle-vector-motion-plan

### PR DESCRIPTION
**Pull request by pazeshun**
Like fetcheus:
https://github.com/jsk-ros-pkg/jsk_robot/blob/master/jsk_fetch_robot/fetcheus/fetch-interface.l#L79
https://github.com/jsk-ros-pkg/jsk_robot/blob/master/jsk_fetch_robot/fetcheus/fetch-interface.l#L82
https://github.com/jsk-ros-pkg/jsk_robot/blob/master/jsk_fetch_robot/fetcheus/fetch-interface.l#L91

### CODE REVIEW

File: jsk_baxter_robot/baxtereus/baxter-interface.l [hunk 1]
```diff
@@ -252,8 +252,8 @@
      (progn
        (unless tm (setq tm 3000))
        (send-super* :angle-vector-motion-plan av :ctype ctype :move-arm move-arm :total-time tm
-                    :start-offset-time start-offset-time :clear-velocities clear-velocities
-                    :use-torso nil args))
+                    :start-offset-time (if start-offset-time start-offset-time start-time)
```
Comment (knorth55) [inline jsk_baxter_robot/baxtereus/baxter-interface.l]: `start-time` is `0`, but is it OK?

Comment (pazeshun) [reply jsk_baxter_robot/baxtereus/baxter-interface.l]: The robot is correctly moved by `(send *ri* :angle-vector (send *baxter* :angle-vector) 1000)` in this configuration.
Default value of `start-offset-time` of `:angle-vector-motion-plan` is also `0`:
https://github.com/jsk-ros-pkg/jsk_pr2eus/blob/0.3.13/pr2eus_moveit/euslisp/robot-moveit.l#L527
 
The only method having non-zero `start-offset-time` as default is `:angle-vector-make-trajectory`:
https://github.com/jsk-ros-pkg/jsk_pr2eus/blob/0.3.13/pr2eus_moveit/euslisp/robot-moveit.l#L455
You added this to follow `:angle-vector-sequence`, right?
https://github.com/jsk-ros-pkg/jsk_pr2eus/commit/0135eacd18d972852d58fca6c476f4fa5db4005a
https://github.com/jsk-ros-pkg/jsk_pr2eus/blob/0.3.13/pr2eus/robot-interface.l#L406

So, what is problem seems that `:angle-vector-sequence` has non-zero `start-offset-time`, which is different from `:angle-vector`...

**Comment by pazeshun** [comment]
I fixed `:angle-vector-sequence` as with `:angle-vector` and added tests about `start-time` and `start-offset-time`.

**Comment by pazeshun** [comment]
Those tests require https://github.com/jsk-ros-pkg/jsk_pr2eus/pull/361, https://github.com/jsk-ros-pkg/jsk_pr2eus/pull/363 and https://github.com/jsk-ros-pkg/jsk_pr2eus/pull/365.
So those tests will fail currently.

**Comment by pazeshun** [comment]
> Those tests require [jsk-ros-pkg/jsk_pr2eus#361](https://github.com/jsk-ros-pkg/jsk_pr2eus/pull/361), [jsk-ros-pkg/jsk_pr2eus#363](https://github.com/jsk-ros-pkg/jsk_pr2eus/pull/363) and [jsk-ros-pkg/jsk_pr2eus#365](https://github.com/jsk-ros-pkg/jsk_pr2eus/pull/365).

They are released, so travis is expected to pass.

**Comment by k-okada** [comment]
something wrong with baxter moveit test
```
[baxtereus.rosunit-baxter_moveit_test/test-baxter-moveit-angle-vector-sequence-fast][passed]                                                                           
[baxtereus.rosunit-baxter_moveit_test/test-baxter-moveit-angle-vector-start-time][FAILURE]
Test:(> tm-diff 5)                                                                              
Trace:"^[0mmstart testing [test-baxter-moveit-angle-vector-start-time]                                                                                
t]                                                                               
X732eec8&gt; :robot-interface-simulation-callback)) lisp::count-"                                                                           
Message:"start-time is ignored. Traj finishes at 0.057"                                                                          
--------------------------------------------------------------------------------                                                                              
[baxtereus.rosunit-baxter_moveit_test/test-baxter-moveit-angle-vector-sequence-start-time][FAILURE]                                                                              
Test:(> tm-diff 5)                                                                             
Trace:"^[0mmstart testing [test-baxter-moveit-angle-vector-sequence-start-time]                                                                              
lback)) lisp::count-).^[0m                                                                             
 testing [test-baxter-moveit"                                                                             
Message:"start-time is ignored. Traj finishes at 0.053"                                                                         
--------------------------------------------------------------------------------
```

**Comment by pazeshun** [comment]
The reason of the test failure seems #1065.
This is fixed in #1066.

**Comment by pazeshun** [comment]
Move to #1068, which is rebased on `origin/master` including #1066

**Pull request closed without merging**